synapse/lib/stormhttp.py

```diff
@@ -198,24 +219,21 @@ async def _httpEasyGet(self, url, headers=None, ssl_verify=True, params=None, ti
         return await self._httpRequest('GET', url, headers=headers, ssl_verify=ssl_verify, params=params,
                                        timeout=timeout, allow_redirects=allow_redirects, )
 
-    async def _httpPost(self, url, headers=None, json=None, body=None, ssl_verify=True, params=None, timeout=300,
-                        allow_redirects=True):
+    async def _httpPost(self, url, headers=None, json=None, body=None, fields=None,
+                        ssl_verify=True, params=None, timeout=300, allow_redirects=True):
```

Comment:
Fields need to be added to the end of the arguments list, otherwise we risk breaking users who are calling this without keyword arguments.

Reply:
fixed